src/k8s-engine/lima.ts

```diff
@@ -1763,14 +1763,25 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   }
 
   async factoryReset(): Promise<void> {
-    const pathsToDelete = [paths.cache, paths.appHome, paths.config, paths.logs];
-
-    if (!pathsToDelete.some( dir => paths.lima.startsWith(dir))) {
+    const promises: Array<Promise<void>> = [];
+    const pathsToDelete = new Set([
+      paths.cache,
+      paths.appHome,
+      paths.altAppHome,
+      paths.config,
+      paths.logs,
+    ]);
+
+    if (!Array.from(pathsToDelete).some(dir => paths.lima.startsWith(dir))) {
```

Comment:
Why do we need this? Is this triggered when someone moves the lima directory elsewhere and replaces it with a symlink?

I'm kind of conflicted about deleting a directory the user has created themselves. It will not interfere with the factory reset, because the symlink got deleted, and the new lima directory will be created in the default place again. So maybe it is better to just display a warning dialog in this case? Or ask for permission to delete this directory as well? Obviously in a different PR, but what do you think?

Reply:
No; `paths.lima` is hard coded, it's just a filter because on mac, `paths.appHome` is a prefix of `paths.lima`.  However, this is _not_ the case on Linux, so we need to remove both.

Reply:
I guess this could use some comment then? 😄